### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by san00**

### CODE REVIEW

File: dist/css/style.css [hunk 1]
```diff
@@ -85,24 +85,25 @@ html {
...
-  flex-wrap: wrap;
+  margin-bottom: calc(1rem * 4);
+  margin-left: calc(1rem * 8);
+  margin-right: calc(1rem * 8);
   -webkit-box-pack: center;
   -ms-flex-pack: center;
   justify-content: center;
-  margin-bottom: calc(1rem * 4);
-  row-gap: 1rem; }
+  -webkit-box-orient: horizontal;
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: If you are not using Autoprefixer remove unnecesary prefixed props

File: dist/css/style.css [hunk 2]
```diff
@@ -85,24 +85,25 @@ html {
...
+  -webkit-box-direction: normal;
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Same as above

File: dist/css/style.css [hunk 3]
```diff
@@ -85,24 +85,25 @@ html {
...
+  -ms-flex-flow: row wrap;
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Same as above

File: dist/index.html [hunk 4]
```diff
@@ -9,44 +9,47 @@
...
     </section>
     <section class="portfolio">
-        <nav class="portfolio__navbar">
-          <a href="" class="portfolio__navbar-links">All</a>
-          <a href="" class="portfolio__navbar-links">lifestyle</a>
-          <a href="" class="portfolio__navbar-links">photo</a>
-          <a href="" class="portfolio__navbar-links">branding</a>
-        </nav>
+      <nav class="portfolio__navbar">
+        <a href="" class="portfolio__navbar-links">All</a>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Where should this links point to?

File: dist/index.html [hunk 5]
```diff
@@ -9,44 +9,47 @@
...
+        <img
           class="portfolio__images-gallery"
           src="images/stephan-valentin-1415671-unsplash.jpg"
           alt="times square, New York"/>
         <img
           class="portfolio__images-gallery"
-          src="images/sole-bicycles-1418884-unsplash.jpg"
-          alt="Bicycle"/>
+          src="images/stephan-valentin-1415671-unsplash.jpg"
+          alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Please name it correctly, like people names.

File: dist/index.html [hunk 6]
```diff
@@ -9,44 +9,47 @@
...
         <img
           class="portfolio__images-gallery"
-          src="images/sole-bicycles-1418884-unsplash.jpg"
-          alt="Bicycle"/>
+          src="images/stephan-valentin-1415671-unsplash.jpg"
+          alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above

File: dist/index.html [hunk 7]
```diff
@@ -9,44 +9,47 @@
...
         <img
-          class="portfolio__images-gallery"
+          class="portfolio__images-gallery photo"
           src="images/stephan-valentin-1415671-unsplash.jpg"
           alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above

File: dist/index.html [hunk 8]
```diff
@@ -55,8 +58,9 @@
...
         <img
           class="portfolio__images-gallery"
-          src="images/sole-bicycles-1418884-unsplash.jpg"
-          alt="Bicycle"/>
+          src="images/stephan-valentin-1415671-unsplash.jpg"
+          alt="times square, New York"
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above.

File: src/index.html [hunk 9]
```diff
@@ -9,44 +9,47 @@
...
     </section>
     <section class="portfolio">
-        <nav class="portfolio__navbar">
-          <a href="" class="portfolio__navbar-links">All</a>
-          <a href="" class="portfolio__navbar-links">lifestyle</a>
-          <a href="" class="portfolio__navbar-links">photo</a>
-          <a href="" class="portfolio__navbar-links">branding</a>
-        </nav>
+      <nav class="portfolio__navbar">
+        <a href="" class="portfolio__navbar-links">All</a>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Where should this empty links point to?

File: src/index.html [hunk 10]
```diff
@@ -9,44 +9,47 @@
...
+        <img
           class="portfolio__images-gallery"
           src="images/stephan-valentin-1415671-unsplash.jpg"
           alt="times square, New York"/>
         <img
           class="portfolio__images-gallery"
-          src="images/sole-bicycles-1418884-unsplash.jpg"
-          alt="Bicycle"/>
+          src="images/stephan-valentin-1415671-unsplash.jpg"
+          alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above.

File: src/index.html [hunk 11]
```diff
@@ -9,44 +9,47 @@
...
         <img
           class="portfolio__images-gallery"
-          src="images/sole-bicycles-1418884-unsplash.jpg"
-          alt="Bicycle"/>
+          src="images/stephan-valentin-1415671-unsplash.jpg"
+          alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above.

File: src/index.html [hunk 12]
```diff
@@ -55,8 +58,9 @@
           alt="times square, New York"/>
         <img
           class="portfolio__images-gallery"
-          src="images/sole-bicycles-1418884-unsplash.jpg"
-          alt="Bicycle"/>
+          src="images/stephan-valentin-1415671-unsplash.jpg"
+          alt="times square, New York"
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above.

**Review by IgnaciodeNuevo — CHANGES_REQUESTED** [review]
Small details :) Good job.

File: dist/index.html [hunk 13]
```diff
@@ -36,31 +36,96 @@
...
-        <img
-          class="portfolio__images-gallery"
-          src="images/stephan-valentin-1415671-unsplash.jpg"
-          alt="times square, New York"
-        />
+          <div class="portfolio__overlay">
+              <img
+                class="portfolio__images-gallery "
+                src="images/stephan-valentin-1415671-unsplash.jpg"
+                alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as the last commit.

File: dist/index.html [hunk 14]
```diff
@@ -36,31 +36,96 @@
...
+                    />
+                  </svg>
+                overlay</a>
+              </div>
+            </div>
+            <div class="portfolio__overlay">
+                <img
+                  class="portfolio__images-gallery "
+                  src="images/stephan-valentin-1415671-unsplash.jpg"
+                  alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as the last commit.

File: dist/index.html [hunk 15]
```diff
@@ -36,31 +36,96 @@
...
+                      />
+                    </svg>
+                  overlay</a>
+                </div>
+              </div>
+          <div class="portfolio__overlay">
+              <img
+                class="portfolio__images-gallery "
+                src="images/stephan-valentin-1415671-unsplash.jpg"
+                alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as the last commit.

File: dist/index.html [hunk 16]
```diff
@@ -36,31 +36,96 @@
...
+                    />
+                  </svg>
+                overlay</a>
+              </div>
+            </div>
+            <div class="portfolio__overlay">
+                <img
+                  class="portfolio__images-gallery "
+                  src="images/stephan-valentin-1415671-unsplash.jpg"
+                  alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as the last commit.

File: dist/index.html [hunk 17]
```diff
@@ -36,31 +36,96 @@
...
+                <div class="portfolio__overlay--effect">
+                  <a href="#" class="icon">
+                    <svg class="icon__link" xmlns="http://www.w3.org/2000/svg">
+                      <path
+                        d="m6.188 8.719c.439-.439.926-.801 1.444-1.087 2.887-1.591 6.589-.745 8.445 2.069l-2.246 2.245c-.644-1.469-2.243-2.305-3.834-1.949-.599.134-1.168.433-1.633.898l-4.304 4.306c-1.307 1.307-1.307 3.433 0 4.74s3.433 1.307 4.74 0l1.327-1.327c1.207.479 2.501.67 3.779.575l-2.929 2.929c-2.511 2.511-6.582 2.511-9.093 0s-2.511-6.582 0-9.093zm6.836-6.836-2.929 2.929c1.277-.096 2.572.096 3.779.574l1.326-1.326c1.307-1.307 3.433-1.307 4.74 0s1.307 3.433 0 4.74l-4.305 4.305c-1.311 1.311-3.44 1.3-4.74 0-.303-.303-.564-.68-.727-1.051l-2.246 2.245c.236.358.481.667.796.982.812.812 1.846 1.417 3.036 1.704 1.542.371 3.194.166 4.613-.617.518-.286 1.005-.648 1.444-1.087l4.304-4.305c2.512-2.511 2.512-6.582.001-9.093-2.511-2.51-6.581-2.51-9.092 0z"
+                      />
+                    </svg>
+                  overlay</a>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Just overlay?

File: dist/index.html [hunk 18]
```diff
@@ -36,31 +36,96 @@
...
+                </div>
+              </div>
+              <div class="portfolio__overlay">
+                  <img
+                    class="portfolio__images-gallery "
+                    src="images/stephan-valentin-1415671-unsplash.jpg"
+                    alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as the last commit.

File: dist/index.html [hunk 19]
```diff
@@ -36,31 +36,96 @@
...
+                  <div class="portfolio__overlay--effect">
+                    <a href="#" class="icon">
+                      <svg class="icon__link" xmlns="http://www.w3.org/2000/svg">
+                        <path
+                          d="m6.188 8.719c.439-.439.926-.801 1.444-1.087 2.887-1.591 6.589-.745 8.445 2.069l-2.246 2.245c-.644-1.469-2.243-2.305-3.834-1.949-.599.134-1.168.433-1.633.898l-4.304 4.306c-1.307 1.307-1.307 3.433 0 4.74s3.433 1.307 4.74 0l1.327-1.327c1.207.479 2.501.67 3.779.575l-2.929 2.929c-2.511 2.511-6.582 2.511-9.093 0s-2.511-6.582 0-9.093zm6.836-6.836-2.929 2.929c1.277-.096 2.572.096 3.779.574l1.326-1.326c1.307-1.307 3.433-1.307 4.74 0s1.307 3.433 0 4.74l-4.305 4.305c-1.311 1.311-3.44 1.3-4.74 0-.303-.303-.564-.68-.727-1.051l-2.246 2.245c.236.358.481.667.796.982.812.812 1.846 1.417 3.036 1.704 1.542.371 3.194.166 4.613-.617.518-.286 1.005-.648 1.444-1.087l4.304-4.305c2.512-2.511 2.512-6.582.001-9.093-2.511-2.51-6.581-2.51-9.092 0z"
+                        />
+                      </svg>
+                    overlay</a>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above.

File: dist/index.html [hunk 20]
```diff
@@ -36,31 +36,96 @@
...
+                  </div>
+                </div>
+                <div class="portfolio__overlay">
+                    <img
+                      class="portfolio__images-gallery "
+                      src="images/stephan-valentin-1415671-unsplash.jpg"
+                      alt="times square, New York"/>
+                    <div class="portfolio__overlay--effect">
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as the last commit.

File: src/index.html [hunk 21]
```diff
@@ -36,31 +36,96 @@
...
-        <img
-          class="portfolio__images-gallery"
-          src="images/stephan-valentin-1415671-unsplash.jpg"
-          alt="times square, New York"
-        />
+          <div class="portfolio__overlay">
+              <img
+                class="portfolio__images-gallery "
+                src="images/stephan-valentin-1415671-unsplash.jpg"
+                alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as the last commit.

File: src/index.html [hunk 22]
```diff
@@ -36,31 +36,96 @@
...
+              <div class="portfolio__overlay--effect">
+                <a href="#" class="icon">
+                  <svg class="icon__link" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                      d="m6.188 8.719c.439-.439.926-.801 1.444-1.087 2.887-1.591 6.589-.745 8.445 2.069l-2.246 2.245c-.644-1.469-2.243-2.305-3.834-1.949-.599.134-1.168.433-1.633.898l-4.304 4.306c-1.307 1.307-1.307 3.433 0 4.74s3.433 1.307 4.74 0l1.327-1.327c1.207.479 2.501.67 3.779.575l-2.929 2.929c-2.511 2.511-6.582 2.511-9.093 0s-2.511-6.582 0-9.093zm6.836-6.836-2.929 2.929c1.277-.096 2.572.096 3.779.574l1.326-1.326c1.307-1.307 3.433-1.307 4.74 0s1.307 3.433 0 4.74l-4.305 4.305c-1.311 1.311-3.44 1.3-4.74 0-.303-.303-.564-.68-.727-1.051l-2.246 2.245c.236.358.481.667.796.982.812.812 1.846 1.417 3.036 1.704 1.542.371 3.194.166 4.613-.617.518-.286 1.005-.648 1.444-1.087l4.304-4.305c2.512-2.511 2.512-6.582.001-9.093-2.511-2.51-6.581-2.51-9.092 0z"
+                    />
+                  </svg>
+                overlay</a>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above.

File: src/index.html [hunk 23]
```diff
@@ -36,31 +36,96 @@
...
+              </div>
+            </div>
+            <div class="portfolio__overlay">
+                <img
+                  class="portfolio__images-gallery "
+                  src="images/stephan-valentin-1415671-unsplash.jpg"
+                  alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as the last commit.

File: src/index.html [hunk 24]
```diff
@@ -36,31 +36,96 @@
...
+                <div class="portfolio__overlay--effect">
+                  <a href="#" class="icon">
+                    <svg class="icon__link" xmlns="http://www.w3.org/2000/svg">
+                      <path
+                        d="m6.188 8.719c.439-.439.926-.801 1.444-1.087 2.887-1.591 6.589-.745 8.445 2.069l-2.246 2.245c-.644-1.469-2.243-2.305-3.834-1.949-.599.134-1.168.433-1.633.898l-4.304 4.306c-1.307 1.307-1.307 3.433 0 4.74s3.433 1.307 4.74 0l1.327-1.327c1.207.479 2.501.67 3.779.575l-2.929 2.929c-2.511 2.511-6.582 2.511-9.093 0s-2.511-6.582 0-9.093zm6.836-6.836-2.929 2.929c1.277-.096 2.572.096 3.779.574l1.326-1.326c1.307-1.307 3.433-1.307 4.74 0s1.307 3.433 0 4.74l-4.305 4.305c-1.311 1.311-3.44 1.3-4.74 0-.303-.303-.564-.68-.727-1.051l-2.246 2.245c.236.358.481.667.796.982.812.812 1.846 1.417 3.036 1.704 1.542.371 3.194.166 4.613-.617.518-.286 1.005-.648 1.444-1.087l4.304-4.305c2.512-2.511 2.512-6.582.001-9.093-2.511-2.51-6.581-2.51-9.092 0z"
+                      />
+                    </svg>
+                  overlay</a>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above.

File: src/index.html [hunk 25]
```diff
@@ -36,31 +36,96 @@
...
+                    />
+                  </svg>
+                overlay</a>
+              </div>
+            </div>
+            <div class="portfolio__overlay">
+                <img
+                  class="portfolio__images-gallery "
+                  src="images/stephan-valentin-1415671-unsplash.jpg"
+                  alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as the last commit.

File: src/index.html [hunk 26]
```diff
@@ -36,31 +36,96 @@
...
+                      />
+                    </svg>
+                  overlay</a>
+                </div>
+              </div>
+              <div class="portfolio__overlay">
+                  <img
+                    class="portfolio__images-gallery "
+                    src="images/stephan-valentin-1415671-unsplash.jpg"
+                    alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as the last commit.

File: src/index.html [hunk 27]
```diff
@@ -36,31 +36,96 @@
...
+                        />
+                      </svg>
+                    overlay</a>
+                  </div>
+                </div>
+                <div class="portfolio__overlay">
+                    <img
+                      class="portfolio__images-gallery "
+                      src="images/stephan-valentin-1415671-unsplash.jpg"
+                      alt="times square, New York"/>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as the last commit.

**Review by IgnaciodeNuevo — CHANGES_REQUESTED** [review]
Almost done, good job San :)

File: dist/index.html (hunk 5 above)
Comment (san00) [reply dist/index.html]: Do you mean remove the numbers?

File: dist/css/style.css (hunk 1 above)
Comment (san00) [reply dist/css/style.css]: I'm using autoprefixer

File: dist/index.html (hunk 5 above)
Comment (IgnaciodeNuevo) [reply dist/index.html]: No like this: "Times Square, New York".

**Review by IgnaciodeNuevo — APPROVED** [review]

**Pull request merged**